external/CapnProto.cmake

```diff
@@ -21,23 +21,34 @@
 
 # Build Cap'n Proto from source.
 #
-# OUTPUTS
+# OUTPUT VARIABLES:
+#
+# CAPNP_STATIC_LIB_TARGET: name of static library target that contains all of
+#                          capnproto library objects.
 #
-# CAPNP_LINK_LIBRARIES: list of capnproto libraries (paths) needed by apps for linking
```

Comment:
capnproto generates 3 static libs that we need: kj, capnp, and capnpc. We now merge and wrap them in  a single `add_library` target, represented simply by `${CAPNP_STATIC_LIB_TARGET}`.